### ISSUE TITLE
Add possibility to exclude columns from shifting

### ISSUE THREAD

**Issue by mpibpc-mroose** [djk2/django-tables2-column-shifter#29]
Given I have a table with let's say an index column. One may want to have this column always visible for whatever reason. Thus it should not be possible to enable or disable this column.

Is this already possible?

**Comment by djk2** [comment]
Sorry for delay. 
It shouldn't be a problem to add this. 
I can add attribute like `column_excluded`  this attribute will be  list(str) type and by default empty.
I will try prepare solution in next few days.

**Comment by djk2** [comment]
column_excluded is available in new  release v2.1.0
New version v 2.1.0 is now available on PyPi
You can install or update it via :

```
pip install -U django-tables2-column-shifter
```
or 
```
pip install -U django-tables2-column-shifter==2.1.0
```

**Comment by mpibpc-mroose** [comment]
Thanks a lot! Works as a dream.
